sources/packages/web/src/services/ApplicationOfferingChangeRequestService.ts

```diff
@@ -128,6 +130,19 @@ export class ApplicationOfferingChangeRequestService {
     );
   }
 
+  /**
+   * Gets the Application Offering Change Request details for ministry view.
+   * @param applicationOfferingChangeRequestId the Application Offering Change Request id.
+   * @returns Application Offering Change Request details.
+   */
+  async getApplicationOfferingDetailsForReview(
+    applicationOfferingChangeRequestId: number,
+  ): Promise<ApplicationOfferingDetailsReviewAPIOutDTO> {
```

Comment:
I remember we talking about having a model which is attached to component and using different DTOs at *Api.ts 

@andrewsignori-aot @ann-aot ?

Reply:
I see the point @dheepak-aot. That is really important especially when the Ministry/Institution/Student are sharing the same component and the component can define a model that would be the combination of the possible returned DTOs.
Here I believe that Ministry and Student components are different and they would not share the endpoint (unless I am wrong).
Also, the DTO on Vue and API are still the same, so we are not impacting that decision.
@dheepak-aot do you believe that a service model would be needed for this situation if there is no adaptation needed since the components are different?

Reply:
I get the point. I believe service model is not essential where there is no adaptation for common component is required.